src/reference/asciidoc/barrier.adoc

```diff
@@ -0,0 +1,87 @@
+[[barrier]]
+=== Thread Barrier
+
+Sometimes, we need to suspend a message flow thread until some other asynchronous even occurs.
```

Comment:
Typo: even**t**
Will be fixed on merge.
With other simple code style polishing.